tests/test_tutorials.py

```diff
@@ -56,6 +56,16 @@ def test_pyam_first_steps(capsys):
     assert errors == []
     assert os.path.exists(os.path.join(tut_path, 'tutorial_export.xlsx'))
 
+    def has_log_output(cell):
+        return (
+            cell['cell_type'] == 'code'
+            and any(
```

Comment:
Charming, the "Best practice" descriptions are cyclic: [W504](https://www.flake8rules.com/rules/W503.html) suggests to use what triggers [W503](https://www.flake8rules.com/rules/W503.html) suggests to use what triggers W504 :).

Since [PEP8](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) suggests to use the latter, i'll disable the W503.